src/software/ai/hl/stp/tactic/chip_tactic_test.cpp

```diff
@@ -2,15 +2,254 @@
 
 #include <gtest/gtest.h>
 
+#include "software/ai/hl/stp/action/chip_action.h"
 #include "software/test_util/test_util.h"
 
-TEST(ChipTacticTest, constructor_test)
+void compareChipActions(std::shared_ptr<ChipAction> chip_action,
+                        std::shared_ptr<ChipAction> expected_chip_action)
+{
+    EXPECT_EQ(expected_chip_action->getChipOrigin(), chip_action->getChipOrigin());
+    EXPECT_EQ(expected_chip_action->getChipDirection(), chip_action->getChipDirection());
+    EXPECT_EQ(expected_chip_action->getChipDistanceMeters(),
```

Comment:
the chip distance is a double, so this should use `EXPECT_DOUBLE_EQ` to help avoid numerical errors when checking for equality

Reply:
Done.